extension/mediator.js

```diff
@@ -19,6 +19,11 @@ window.addEventListener("load", function () {
   });
 });
 
+const getOrigin = () => {
+    return window.origin !== null
```

Comment:
```suggestion
    return window.origin !== "null"
```

Here is a test page; it has the `Content-Security-Policy: sandbox allow-scripts` header: https://robwu.nl/cors-anywhere.html

You can also test this even if the page doesn't set the header by specifying `<iframe src="https://example.com/" sandbox=""></iframe>`, as long as the sandbox attribute does not contain "allow-same-origin".

For documentation on the sandbox property, see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/sandbox

P.S. Since the value of this is a constant, you may as well change this to `const THIS_ORIGIN = window.origin != "null" ? window.origin : location.origin;`

Reply:
fixed